### DISCUSSION TITLE
MultiSchema support for mongo connector

### DISCUSSION THREAD

**Discussion by mpgalaxy** [prisma/prisma#18268] [Q&A]
Hi,

I'm migrating our prisma1 services to prisma and in one service I need support for three different models with mongodb. 
Is there a way to do that ? I can't use nested documents for this purpose.
When I try to use the preview feature, prisma generate tells me that it is not supported (yet?), using prisma 4.10.1:
```prisma
generator client {
  provider = "prisma-client-js"
  previewFeatures = ["multiSchema"]
}

datasource db {
  provider = "mongodb"
  url      = env("DATABASE_URL")
  schemas  = ["EmailTemplates", "EmailLog"]
}
```
Resulting error:
```
Error: Prisma schema validation - (get-dmmf wasm)
Error code: P1012
error: The `schemas` property is not supported on the current connector.
  -->  schema.prisma:9
   | 
 8 |   url      = env("DATABASE_URL")
 9 |   schemas  = ["EmailTemplates", "EmailLog"]
   | 

Validation Error Count: 1
[Context: getDmmf]

Prisma CLI Version : 4.10.1
```

It would be really awesome if we had this feature for mongodb, too. ;-)

UPDATE:
Interestingly this works, I had a typo in the name of a nexus objectType and the other models weren't generated. The previewFeature isn't needed, this works out of the box with mongodb...very nice! :)

BTW: Is there a way of handling migrations / seedings with mongodb ? In prisma1 we used Types SeededData and Migration, prisma1 took care of it. Is there a pendant to this in prisma or do I have to handle this manually (querying and setting entries in these collections) ?

**Comment by ludralph** [comment]
Hello @mpgalaxy 👋 

Prisma Migrate does not apply to MongoDB. It is recommended to use `db push` when working with MongoDB. See this [section](https://www.prisma.io/docs/concepts/components/prisma-migrate#:~:text=Does%20not%20apply%20for%20MongoDB%0AInstead%20of%20migrate%20dev%20and%20related%20commands%2C%20use%20db%20push%20for%20MongoDB.) of the documentation. You would have to handle seeding manually, as you've suggested.

**Reply by ludralph** [reply]
Hi @mpgalaxy 👋 

Do you have any more questions around this issue? If no, it would be really helpful if you could quickly **mark this Discussion as _answered_** to indicate that your issue has been resolved. 

In case you do have any outstanding questions, feel free to let us know and we'd be happy to help!

**Reply by ludralph** [reply]
Hi @mpgalaxy 

You may have missed my previous comments. Do let us know if you have any further questions or kindly mark this discussion as answered.
